tracer/src/Datadog.Trace/Iast/IastUtils.cs

```diff
@@ -71,9 +54,20 @@ public static int GetHashCode<T>(T value)
     }
 
     private static int GetHash<T>(T element)
-       => ((element is Array array)
-            ? GetHashCodeForArray(array, 1)
```

Comment:
arrays dont need a hascode anymore?

Reply:
Yes, not used in calculating the over all hash used by the backend